unity-renderer/Assets/UnityGLTF/Scripts/GLTFComponent.cs

```diff
@@ -159,13 +159,12 @@ void ApplySettings(Settings settings)
         private void OnFail_Internal(Exception obj)
         {
             if (state == State.FAILED)
-            {
                 return;
-            }
 
             state = State.FAILED;
 
-            CoroutineStarter.Stop(loadingRoutine);
+            if (loadingRoutine != null)
```

Comment:
This is actually not neeed, `CoroutineStarter` handles `null` references gracefully.

Reply:
Changed!